tools/wake-format/formatter.h

```diff
@@ -356,6 +369,7 @@ struct Formatter {
   Formatter(Action a) : action(a) {}
 
   Formatter<SeqAction<Action, ConsumeWhitespaceAction>> consume_wsnl() { return {{action, {}}}; }
+  Formatter<SeqAction<Action, CopyWhitespaceAction>> copy_wsnl() { return {{action, {}}}; }
```

Comment:
Not now but I think we need to rethink our handling of white space and newlines (and probably comments) a bit. I think the different types are proliferating and its becoming unwieldy.